CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# 2.14.0 / 2024-09-24
+
+* [FEATURE] Add stop and start APIs for Session Replay. See [#2169](https://github.com/DataDog/dd-sdk-android/pull/2169)
```

Comment:
for the readability it is better to group categories: first `FEATURE`, then `IMPROVEMENT`, then `MAINTENANCE`